content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -29,7 +29,7 @@ Our current process is as follows:
 
   - Engineers, designer, PM and EM **_propose_ issues to be worked on** the week _before_ the mid-iteration [Monday sync](#weekly-sync) by adding them to the next iteration milestone and setting "Status" to "Proposed". This is done considering our [higher-level goals and roadmap](../../../../../strategy-goals/strategy/code-graph/code-insights/index.md).
 
-    - The issue should also have its **_Estimate_ column filled out**, so that it can be evaluated whether it fits into the iteration. If the proposer lacks the information to estimate the issue, they reply on the issue in GitHub or raise it in our Slack channel to get the missing information or get an estimate from the appropiate person. Teammates may also discuss this in ad-hoc synchronous meetings if beneficial. An assignee may also already volunteer or be proposed, but this may still be changed at the [Monday sync](#weekly-sync) to distribute workload.
+    - The issue should also have its **_Size_ column filled out**, so that it can be evaluated whether it fits into the iteration. If the proposer lacks the information to estimate the issue, they reply on the issue in GitHub or raise it in our Slack channel to get the missing information or get an estimate from the appropiate person. Teammates may also discuss this in ad-hoc synchronous meetings if beneficial. An assignee may also already volunteer or be proposed, but this may still be changed at the [Monday sync](#weekly-sync) to distribute workload. See below for information on issue sizing.
```

Comment:
I know you didn't add this, but as I read this I find it a little confusing. Why would someone be proposing an issue if they lack the context on the issue? In the case of a PM or EM, that may make sense, but in general I'd expect someone who doesn't have the context to not be able to advocate for an issue.

Reply:
I think this is fine personally. I could see a description sounding like something you want to see happen without knowing the implementation details enough to size it. This point here helps to point out that you should got _get_ context before trying to propose an issue to be worked on.